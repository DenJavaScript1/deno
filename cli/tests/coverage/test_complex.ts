const dir = await Deno.makeTempDir();

const test = await Deno.run({
  cmd: [
    Deno.execPath(),
    "test",
    "--unstable",
    "--coverage=" + dir,
    "coverage/complex_test.ts",
  ],
  stdout: "inherit",
  stderr: "inherit",
});

await test.status();

const coverage = await Deno.run({
  cmd: [
    Deno.execPath(),
    "coverage",
    "--unstable",
    dir,
  ],
  stdout: "inherit",
  stderr: "inherit",
});

await coverage.status();
