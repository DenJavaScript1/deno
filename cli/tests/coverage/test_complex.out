Check [WILDCARD]/test_complex.ts
Check [WILDCARD]/$deno$test.ts
running 1 tests
test complex ... ok ([WILDCARD])

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out ([WILDCARD])

cover [WILDCARD]/coverage/complex.ts ... 67.742% (21/31)
  40 | export function unused(
  41 |   foo: string,
  42 |   bar: string,
  43 |   baz: string,
-----|-----
  45 |   return complex(
  46 |     foo,
  47 |     bar,
  48 |     baz,
  49 |   );
  50 | }
